CMakeLists.txt

```diff
@@ -3470,6 +3444,51 @@ if(FFMPEG)
   target_include_directories(mixxx-lib PUBLIC "${FFMPEG_INCLUDE_DIRS}")
 endif()
 
+# STEM file support
+default_option(STEM "STEM file support" "FFMPEG_FOUND;FFMPEG")
+if (STEM)
+  if(NOT FFMPEG)
+    message(FATAL_ERROR "FFMPEG was not enabled")
+  endif()
+  target_compile_definitions(mixxx-lib PUBLIC __STEM__)
+  target_compile_definitions(mixxx-test PUBLIC __STEM__)
+  target_sources(mixxx-test PUBLIC
+    src/test/stemtest.cpp
+    src/test/steminfotest.cpp
+  )
+  target_sources(mixxx-lib PRIVATE
+    src/sources/soundsourcestem.cpp
+    src/track/steminfoimporter.cpp
+    src/track/steminfo.cpp # TODO precompiled header?
```

Comment:
Yes, please add track/steminfo.h to the list of precompiled headers. In the follow-up PRs it's included at various places an makes therefore sense.

Reply:
Please add the header to the precompiled header list

Reply:
I haven't figured out how to make that conditionally yet and will add once I have a solution. Happy to do right away if you have a recommended way.

Reply:
Looks like `target_precompile_headers` adds headers and don't override the list as I thought. Added now